### DISCUSSION TITLE
What's the Configuration command?

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#2100] [Unsupported Versions] [answered]
I installed using the 2.3.2 ISO, and when I logged into the CLI it automatically started the Configuration.  I answered a few of the questions but made a mistake about half way through, so I chose Cancel for the rest of the questions since there was no Back option.  This cancelled the Configuration as expected, but I can't figure out what command to enter to restart it.  I've reviewed all the Installation and Configuration instructions but it doesn't specify any command to enter, it seems to jump from Installation to Configuration without specifying how to trigger it manually if something goes wrong.

What's the command?

**Comment by sleepingbel** [comment, marked as answer]
Hello,

The command is "sudo bash so-setup-network"

You find it in the documentation
[https://docs.securityonion.net/en/2.3/installation.html](https://docs.securityonion.net/en/2.3/installation.html)

Regards
Bart

**Reply by innovate-support** [reply]
It's NOT explicitly stated in the docs.

To re-setup everything, even IP info:
`sudo bash /home/YOURUSER/SecurityOnion/setup/so-setup iso`

To re-setup SO stuff but leave the IP address alone.
`sudo bash /home/YOURUSER/SecurityOnion/setup/so-setup network`
or
`sudo bash /home/YOURUSER/SecurityOnion/so-setup-network`

**Reply by dougburks** [reply]
I've updated the Installation page:
https://docs.securityonion.net/en/2.3/installation.html
